src/lib_smart_leds.rs

```diff
@@ -91,6 +92,17 @@ where
             phantom: Default::default(),
         })
     }
+
+    /// Create a new driver wrapper.
+    pub fn new_with_rmt_driver<C: RmtChannel>(
```

Comment:
I believe that the generic parameter and its bound `<C: RmtChannel>` can be removed.

Reply:
:heavy_check_mark: